proxyservice/jenkins.go

```diff
@@ -116,3 +116,16 @@ func (j *Jenkins) TriggerDockerhubJob(data *DockerHubWebhookData) error {
 	params.Set("RawJSON", string(rawJSON))
 	return j.TriggerJob(path, params)
 }
+
+func (j *Jenkins) TriggerHgJob(repoUrl string, head string, data *HgMessage) error {
+	rawJSON, err := json.Marshal(data)
+	if err != nil {
+		return fmt.Errorf("Error marshaling data: %v", err)
+	}
+	path := path.Join("/job/hgmo/job", repoUrl)
```

Comment:
What will the `repoUrl` look like?

Reply:
`repoUrl` was the wrong thing (thanks for catching this). I've replaced it with `repoPath` which will look like `ci/ci-admin` or `mozilla-central` or `releases/mozilla-beta`.

Reply:
We should validate it as well before calling jenkins. We should check that it only includes: `[A-z0-9-/]` and then we need to split that on `/` and add job between them all.